src/main/java/at/ac/fhcampuswien/fhmdb/state/DefaultState.java

```diff
@@ -0,0 +1,19 @@
+package at.ac.fhcampuswien.fhmdb.state;
+
+import at.ac.fhcampuswien.fhmdb.controller.HomeController;
+
+import java.util.Comparator;
+
+public class DefaultState extends State{
```

Comment:
Do we actually need a default state, seeing as it's the same as the desc state?

Reply:
From my point of view we could remove it, but the assignment says we got three states. So I am not sure about it.

Reply:
That's true, we need the default state, but I would remove the line `homeController.movies.sort(Comparator.naturalOrder());` from this default state, as we do not want the list to be sorted when we first load it from the API.
In this case, please disregard my reply on the `DESCState.java`  class.